.github/workflows/installer.yml

```diff
@@ -72,10 +74,10 @@ jobs:
       # tar and gzip are needed by the actions/checkout below.
       run: yum install -y --allowerasing tar gzip ${{ matrix.deps }}
       if: |
-        contains(matrix.image, 'centos')
-        || contains(matrix.image, 'oraclelinux')
-        || contains(matrix.image, 'fedora')
-        || contains(matrix.image, 'amazonlinux')
+        contains(matrix.image, 'centos') ||
```

Comment:
No real objection, but why this change?

Reply:
Yeah, autoformat is fine. I found it easier to read before, but the value of having it autoformatted is probably worth it.